strings/language/pirate.txt

```diff
@@ -0,0 +1,226 @@
+are@=be
+am@=be
+ahead@=bow
+front@=bow
+behind@=aft
+back@=aft
```

Comment:
I'd probably either use fore/aft or bow/stern, not bow/aft

Reply:
Also could map North/East/South/West to Fore/Starboard/Aft/Port

Reply:
but NESW are absolute directions, and Fore/Starboard/Aft/Port are relative!

Reply:
I changed the cardinal directions! and went with bow/stern. I like the idea of people trying to cheat the directional system by using n/e/s/w but still using the pirate-y speak